cxx-qt/src/lib.rs

```diff
@@ -40,6 +40,25 @@ pub fn bridge(_attr: TokenStream, input: TokenStream) -> TokenStream {
     extract_and_generate(module, &cpp_namespace_prefix)
 }
 
+/// A macro which describes that an enum the definition of signals for a QObject
```

Comment:
We're either missing an `is` here, or we need a small rephrasing:
`A macro which describes that an enum defines the signals for a QObject`.

Reply:
#186